setup.py

```diff
@@ -53,6 +53,7 @@
         "strawberryfields.backends.tfbackend",
         "strawberryfields.backends.fockbackend",
         "strawberryfields.backends.gaussianbackend",
+        "strawberryfields.future.apps"
```

Comment:
Are there any changes to `requirements` above that should be added to this file?

Reply:
It depends, the `requirements.txt` for the applications layer is here:
https://github.com/XanaduAI/glassonion/blob/master/requirements.txt

It has things like `pytest`, which is not included in SF. However, I put things like `pytest` in the apps `requirements.txt` because I was following the style of exactly constructing a reproducible environment. Instead, it looks like the SF `requirements.txt` is not quite as strict as this. In this case, SF definitely has all the bare essentials needed for the applications layer (which is really `networkx` and `numpy`).

Reply:
The current `requirements.txt` file in Strawberry Fields is not ideal. Ideally, we should be including pinned versions of the required packages we are using in development (i.e., `numpy==1.17.0`).

Some guides recommend using `pip freeze > requirements.txt`, but this is problematic, and can lead to broken installations (we had this issue early on with the first few SF releases).

In terms of packages required for testing rather than adding them to the existing `requirements.txt`, I would prefer something like `requirements-test.txt`, which is pretty common (but not ubiquitous).